internal/generator/vector/output/otlp/otlp_all.toml

```diff
@@ -24,7 +24,8 @@ source = '''
   resource.attributes = append( resource.attributes,
       [{"key": "k8s.pod.name", "value": {"stringValue": get!(.,["kubernetes","pod_name"])}},
       {"key": "k8s.container.name", "value": {"stringValue": get!(.,["kubernetes","container_name"])}},
-      {"key": "k8s.namespace.name", "value": {"stringValue": get!(.,["kubernetes","namespace_name"])}}]
+      {"key": "k8s.namespace.name", "value": {"stringValue": get!(.,["kubernetes","namespace_name"])}},
+       {"key": "log.iostream", "value": {"stringValue": get!(.,["kubernetes","container_iostream"])}}]
```

Comment:
We can revert this change since IOStream is part of the log entry where this grouping is a way to define the overall resource which is the container workload